test/it/uk/gov/hmrc/bindingtariffclassification/component/BaseFeatureSpec.scala

```diff
@@ -34,6 +34,8 @@ abstract class BaseFeatureSpec extends FeatureSpec
 
   protected lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
 
+  protected lazy val api_token_key = "X-Api-Token"
```

Comment:
I suggest to use camel case (for consistency with the other code, but also because most Scala developers use it)

Reply:
I think IntelliJ picked the name when we extracted variable. Changed it to camel case